src/onepasswordconnectsdk/client.py

```diff
@@ -49,9 +50,9 @@ def get_file(self, file_id: str, item_id: str, vault_id: str):
                 f"Unable to retrieve item. Received {response.status_code}\
                                      for {url} with message: {response.json().get('message')}"
             )
-        return self.serializer.deserialize(response.content, "File")
+        return self.deserialize(response.content, "File")
```

Comment:
The `Client` and `AsyncClient` class methods currently do not use the `deserialize` method defined in their respective classes, and instead use `serializer.deserialize`. Since functionally both approaches should be identical, I agree that using `deserialize` would make the code cleaner, but the `AsyncClient` methods also use `serializer.deserialize`, and we should maintain consistency in the codebase. 

I would recommend reverting this, and all other relevant instances in this file, back to `self.serializer.deserialize` for now, to maintain consistency in the codebase. Then in a follow-up MR, we can consider whether we want to refactor the `Client` and `AsyncClient` classes to use `deserialize`, or otherwise delete the unused method.

Reply:
Hmm, ok, what a mess :D I'll revert that!

Reply:
I don't think you missed it, I think I reintroduced this in my last rebase to get the signed commits. It's a little confusing when there's two ways to do the exact same thing present in the file. Should have seen that it was being changed with the merge to master, though!